front/src/applications/operationalStudies/components/Scenario/Timetable.jsx

```diff
@@ -0,0 +1,194 @@
+    const params = {
+      timetable: trainDetail.timetable,
+      path: trainDetail.path,
+      schedules: [],
+    };
+    let actualTrainCount = 1;
+    for (let nb = 1; nb <= trainCount; nb += 1) {
+      const newTrainDelta = 60 * trainDelta * nb;
+      const newOriginTime = train.departure + newTrainDelta;
+      const newTrainName = trainNameWithNum(trainName, actualTrainCount, trainCount);
+      params.schedules.push({
+        departure_time: newOriginTime,
+        initial_speed: trainDetail.initial_speed,
+        labels: trainDetail.labels,
+        rolling_stock: trainDetail.rolling_stock,
+        train_name: newTrainName,
+        allowances: trainDetail.allowances,
+        speed_limit_composition: trainDetail.speed_limit_composition,
+      });
+      actualTrainCount += trainStep;
+    }
+    try {
+      await post(`${trainscheduleURI}standalone_simulation/`, params);
+      getTimetable();
+      dispatch(
+        setSuccess({
+          title: t('osrdconf:trainAdded'),
+          text: `${trainName}`,
+        })
+      );
+    } catch (e) {
+      console.log('ERROR', e);
+      dispatch(
+        setFailure({
+          name: e.name,
+          message: e.message,
+        })
+      );
+    }
+  };
+
+  const selectPathProjection = async (train) => {
+    if (train) {
+      dispatch(
+        updateSelectedProjection({
+          id: train.id,
+          path: train.path,
+        })
+      );
+    }
+  };
+
+  useEffect(() => {
+    if (debouncedTerm !== '' && departureArrivalTimes) {
+      setTrainsList(departureArrivalTimes.filter((train) => train.name.includes(debouncedTerm)));
+    } else {
+      setTrainsList(departureArrivalTimes);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [departureArrivalTimes, debouncedTerm]);
+
+  return (
+    <div className="scenario-timetable">
+      <div className="scenario-timetable-toolbar">
+        <div className="">{t('trainCount', { count: trainsList ? trainsList.length : 0 })}</div>
+        <div className="flex-grow-1">
+          <InputSNCF
+            type="text"
+            id="scenarios-filter"
+            name="scenarios-filter"
+            value={filter}
+            onChange={(e) => setFilter(e.target.value)}
+            placeholder={t('filterPlaceholder')}
+            whiteBG
+            noMargin
+            unit={<i className="icons-search" />}
+            sm
+          />
+        </div>
+        <button
+          className="btn btn-primary btn-sm"
+          type="button"
+          onClick={() => setDisplayTrainScheduleManagement(MANAGE_TRAIN_SCHEDULE_TYPES.add)}
+        >
+          <FaPlus />
+        </button>
+      </div>
+      <div className="scenario-timetable-trains">
+        {trainsList
+          ? trainsList.map((train, idx) => (
+              <TimetableTrainCard
+                train={train}
+                key={nextId()}
+                selectedTrain={selectedTrain}
```

Comment:
You only use selectedTrain in TimetableTrainCard to do a selectedTrain === idx , so just pass a boolean, otherwise all the components will be rehydrated whithout necessity

Reply:
YES ! Agree at 100%, it was the objective with new scenario API (timetable will disappear), so I will let as this now, and new work will be moved outside.

Reply:
Yep, done with cc561b1